example_configs/cassandra.yml

```diff
@@ -0,0 +1,15 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+rules:
+- labels:
+    address: "$2"
+  name: cassandra_$1_$3
+  pattern: org.apache.cassandra.metrics<type=(Connection|Streaming), scope=(S*), name=(S*)><>(Count|Value)
```

Comment:
From a readability standpoint pattern, name, labels would make more sense in the examples.